lib/control-app.js

```diff
@@ -25,10 +41,15 @@ module.exports = function(config) {
 
     var build;
     // default builder is browserify which we provide
-    config.builder = config.builder || '../lib/builder-browserify';
+    config.builder = config.builder || defaultBuilder;
```

Comment:
Agreed on this.

I'm fine adding another "layer" on the builder API. I don't think many people are using it right now since it's not documented and it should be trivial to wrap any existing builders to the new API.

Looks great otherwise! :)